mautic/mautic
# Add id HTML attribute for buttons based on defined action

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR makes default page buttons (like Edit, Close) have an individual ID based on the configured action via PHP or Twig.
In a practical scenario, makes easier to create new keyboard shortcuts targeting specific elements based on their id.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > Create new > Save & Close > Use your Dev tools to check if you have an id for the edit button, for example.
